### PR TITLE
[ADD]Added test cases in assignment and school module.

### PR DESCRIPTION
**Pull request by KrishnaPrajapati-SerpentCS**

### CODE REVIEW

File: assignment/__openerp__.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
  'depends': ['school'],
  'data': ['views/homework_view.xml', 'security/ir.model.access.csv'],
  'demo': ['demo/assignment_demo.xml'],
- 'test': ['test/assignment.yml'],
+#  'test': ['test/assignment.yml'],
  'installable': True,
```
Comment (JayVora-SerpentCS) [inline assignment/__openerp__.py:16]: remove test line

File: assignment/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import test_assignment
```
Comment (JayVora-SerpentCS) [inline assignment/tests/__init__.py:4]: licesne text missing

File: assignment/tests/test_assignment.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+        self.res_users_model = self.env['res.users']
+        self.partner_model = self.env['res.partner']
+        self.company = self.env.ref('base.main_company')
+        self.partner1 = self.env.ref('base.res_partner_1')
+
+        print "self.res_users_model", self.res_users_model
+        print "self.company  ::::", self.company
+        print "self.partner1 :::::", self.partner1
+
```
Comment (JayVora-SerpentCS) [inline assignment/tests/test_assignment.py]: Prints mjst be removed

**Comment by KrishnaPrajapati-SerpentCS** [comment]
Ok sir
All issues are solved

**Pull request merged**